west.yml

```diff
@@ -99,7 +99,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: fb936182435a1ae90fafbb8840e70a45d6f99a4e
+      revision: 121c6fa46dd7f6ee9852751da0357a41ba182a1e
```

Comment:
This is no longer head of master in nrfxlib.
Superseded by: https://github.com/nrfconnect/sdk-nrf/pull/4354

Reply:
Thanks! I will not rebase that nor my original PR.

@greg-fer This one can be closed when #4354 gets in. There is no point in duplicating tests.